test/connectivity/README.md

```diff
@@ -69,3 +72,106 @@ Network offline/online should not affect this test result since Edge hub stores
 6. When DeploymentTester2 starts up, it will get all environment variables and report to Test Result Coordinator (#5).
 
 When network goes offline, route#3 will not happen.  But once the network comes back online, Edge Agent should be able to get the latest updated deployment and deploy DeploymentTester2.
+
+#### Cloud to Module Direct Method Test
+
+![Connectivity Test Diagram](./images/ConnectivityTest_C2M_DirectMethod.png)
+
+The Direct Method Connectivity Test is designed to test direct method resilience against network connectivity issue. 
+##### Test Specific Component:
```

Comment:
It is good to have standard sections - Test specific component and test mechanism/flow.  Please create a task to update Messaging test and deployment test to follow.

Reply:
What PBI should I put this under?

Reply:
if not pbi fit, just create a bug which to align section format.